### ISSUE TITLE
microsoft edge字体无法恢复默认

### ISSUE THREAD

**Issue by LKoiJZ** [Tatsu-syo/noMeiryoUI#62]
开发者你好，选择恢复默认字体后，microsoft edge字体无法恢复默认，依旧是修改后的字体，经过重启电脑以及修复edge均无法使字体恢复默认

系统：Windows11 22H2 22621.963
Nomeiryoui版本：3.1.0
语言环境：简体中文

**Comment by Tatsu-syo** [comment]
No!! Meiryo UI changes system font settings, and handling is depended on applications.
Microsoft Edge is weird application.
In my environments, system fonts are applied on some machines, but another machines are not.
If you can, create new profile on Microsoft Edge.

**Comment by LKoiJZ** [comment]
明白，我会进行实践。目前使用Tampermonkey脚本大致恢复原先网页文字的清晰度，日常使用没有太大问题
